scientific/src/commonMain/kotlin/unit/DimensionlessUnit.kt

```diff
@@ -0,0 +1,108 @@
+ * or a quantity of dimension one,
+ * with a corresponding constant of measurement in the SI of the constant one (or 1), which is not explicitly shown.
+ *
+ * "
+ * 2.3.3 Dimensions of quantities
+ * ...
+ * There are quantities Q for which the defining equation is such that all of the dimensional
+ * exponents in the equation for the dimension of Q are zero.
+ * This is true in particular for any quantity that is defined as the ratio of two quantities of the same kind.
+ * For example, the refractive index is the ratio of two speeds and the relative permittivity is
+ * the ratio of the permittivity of a dielectric medium to that of free space.
+ * Such quantities are simply numbers. The associated constant is the constant one, symbol 1,
+ * although this is rarely explicitly written (see 5.4.7).
+ * ...
+ * The constant one is the neutral element of any system of units – necessary and present automatically.
+ * There is no requirement to introduce it formally by decision.
+ * Therefore, a formal traceability to the SI can be established through appropriate,
+ * validated measurement procedures.
+ *
+ * 5.4.7 Stating quantity values being pure numbers
+ *
+ * As discussed in Section 2.3.3, values of quantities with constant one, are expressed simply as numbers.
+ * The constant symbol 1 or constant name “one” are not explicitly shown.
+ * SI prefix symbols can neither be attached to the symbol 1 nor to the name “one”,
+ * therefore powers of 10 are used to express particularly large or small values.
+ *
+ * Quantities that are ratios of quantities of the same kind (for example length ratios and amount fractions)
+ * have the option of being expressed with units (m/m, mol/mol) to aid the understanding of
+ * the quantity being expressed and also allow the use of SI prefixes, if this is desirable (μm/m, nmol/mol).
+ * Quantities relating to counting do not have this option, they are just numbers.
+ *
+ * The internationally recognized symbol % (percent) may be used with the SI.
+ * When it is used, a space separates the number and the symbol %.
+ * The symbol % should be used rather than the name “percent”.
+ * In written text, however, the symbol % generally takes the meaning of “parts per hundred”.
+ * Phrases such as “percentage by mass”, “percentage by volume”, or “percentage by amount of substance” shall not be used;
+ * the extra information on the quantity should instead be conveyed in the description and symbol for the quantity.
+ * "
+ * Source:  "SI Brochure: The International System of Units, 9th Edition".
+ */
+@Serializable
+
+sealed class Dimensionless : ScientificUnit<PhysicalQuantity.Dimensionless>, MetricAndImperialScientificUnit<PhysicalQuantity.Dimensionless>
+
+val DimensionlessUnits: Set<Dimensionless> get() = setOf(
+    One,
+    Percent,
+    Permill
+)
+
+object One : Dimensionless(), MetricBaseUnit<MeasurementSystem.MetricAndImperial, PhysicalQuantity.Dimensionless> {
```

Comment:
Mark as Serializable please (same for other units)

Reply:
done